workspace/exercise.go

```diff
@@ -49,3 +59,8 @@ func (e Exercise) HasMetadata() (bool, error) {
 	}
 	return false, err
 }
+
+// NewDocument creates a document relative to the exercise.
+func (e Exercise) NewDocument(file string) Document {
```

Comment:
Method on this type really feels like it should be a single function or just deleted as workspace.NewDocument does the same thing.

Reply:
I'll go ahead and make the change--I really don't have a preference one way or the other.